### DISCUSSION TITLE
Multiple Apps on one Dokku Server?

### DISCUSSION THREAD

**Discussion by oneCodeScholar** [dokku/dokku#4542] [Q&A]
I'm trying to run multiple apps on one Dokku server each with their own domain name. Can this be done? When I try to do it now I can only see one site. I have two dokku apps, each with the same build. I have the ports mapped to 3000 on one and 3001 on the other in the dokku port mapping. When I deploy the app they get built using the 3000 port on both it seems. Running dokku logs for the apps it shows: "ready - started server on 0.0.0.0:3000, url: http://localhost:3000"  Only one site show, the first site I created. How can I fix this? I've tried adding a port number to the package.json "build" script but it's not having the desired effect. Do I need to configure a webpack.config file? Or is it best to have only just one app per Dokku server? Thanks!

**Comment by josegonzalez** [comment]
A single server can host multiple apps. Can you post the output of `dokku report $APP` for each app?

**Reply by jrneal1** [reply]
Can I pick up where @oneCodeScholar left off?  I'm having the same issue - two apps but can only get to one.  The dash.txt is the output for the one that is accessible.  flask.txt is not  
[dash.txt](https://github.com/dokku/dokku/files/8980663/dash.txt)
[flask.txt](https://github.com/dokku/dokku/files/8980664/flask.txt)

**Reply by josegonzalez** [reply]
@jrneal1 can create a ticket for your issue? Bugs are better reported as bugs vs in discussions :)
